src/options.html

```diff
@@ -47,6 +47,9 @@ <h3 data-i18n-message-id="firefoxAccountsSync"></h3>
           <span class="bold" data-i18n-message-id="enableSync"></span>
         </label>
         <p><em data-i18n-message-id="enableSyncDescription"></em></p>
+        <br>
+        <input id="syncExcludeRegExp">
```

Comment:
I got a request to add a placeholder for the input element. I guess something like "Insert a regular expression" should be good enough.

Reply:
I think we should also have a label for the input (e.g. "Container exclusion") just to make sure it's as accessible as possible.

Reply:
Had to add some CSS to make it look OK with the combination of label and input box. Adjusted the i18n PR to include the label string. Also added the placeholder as plain sting for now.

Reply:
I found how.
```
        <input id="syncExcludeRegExp"
               data-i18n-attribute="placeholder"
               data-i18n-attribute-message-id="syncInput">
```
It only supports one attribute tho but if you want to update it that's the way.